styles.css

```diff
@@ -0,0 +1,91 @@
+h2 {
+  font-family: "Playfair Display SC", serif;
+  font-size: 42px;
+  letter-spacing: 2px;
+  text-transform: uppercase;
+
+  line-height: 5px;
+  color: white;
+}
+nav {
+  font-family: "Open Sans", sans-serif;
+  font-style: italic;
+  font-size: 16px;
+  line-height: 10px;
+  word-spacing: 5px;
+  color: gray;
+}
+a {
+  color: rgb(75, 75, 75);
+}
+a:link {
+  text-decoration: none;
+}
+a:visited {
+  text-decoration: none;
+}
```

Comment:
You can put the `text-decoration` rule on the default `a` element and then override it _only_ on hover (as you've done on line 28):

```suggestion
a {
  color: rgb(75, 75, 75);
  text-decoration: none;
}
```